src/pages/test-report/components/controlled-stop/__tests__/controlled-stop.spec.ts

```diff
@@ -438,95 +440,50 @@ describe('ControlledStopComponent', () => {
       fixture.detectChanges();
       expect(drivingFaultsBadge.count).toBe(1);
     });
-  });
-
-  describe('Competency button effects', () => {
-    it('should have added no classes to the competency button', () => {
-      const competencyButton = fixture.debugElement.query(By.css('.competency-button'));
-      expect(competencyButton.nativeElement.className).toEqual('competency-button');
-    });
 
-    it('should add the activated class when the button is pressed', () => {
-      component.onTouchStart('competency');
+    it('should pass the allow ripple value to the competency button component', () => {
       fixture.detectChanges();
-      const button = fixture.debugElement.query(By.css('.competency-button'));
+      const competencyButton = fixture.debugElement.query(By.css('competency-button'))
+        .componentInstance as CompetencyButtonComponent;
+      component.allowRipple = false;
 
-      expect(button).toBeDefined();
-      expect(button.nativeElement.className).toContain('activated');
-      expect(component.touchState).toBeTruthy();
-    });
-
-    it('should remove the activated class after a specified delay when the button is not pressed', (done) => {
-      component.onTouchEnd('competency');
       fixture.detectChanges();
-      const button = fixture.debugElement.query(By.css('.competency-button'));
-      setTimeout(
-        () => {
-          fixture.detectChanges();
-
-          expect(button).toBeDefined();
-          expect(button.nativeElement.className).not.toContain('activated');
-          expect(component.touchState).toBeFalsy();
-          done();
-        },
-        component.touchStateDelay);
+      expect(competencyButton.ripple).toBeFalsy(1);
     });
+  });
+
+  describe('Competency button ripple effects', () => {
 
-    it('should add the ripple effect animation css class', () => {
-      component.addOrRemoveFault(true);
+    it('should allow the ripple effect when no faults exist', () => {
       fixture.detectChanges();
-      const button = fixture.debugElement.query(By.css('.competency-button'));
 
-      expect(button).toBeDefined();
-      expect(button.nativeElement.className).toContain('ripple-effect');
+      expect(component.allowRipple).toBeTruthy();
     });
 
-    it('should remove the ripple effect animation css class within the required time frame', (done) => {
-      component.addOrRemoveFault(true);
+    xit('should not allow the ripple effect when faults do exist', () => {
```

Comment:
skipped test?

Reply:
Yes, Tom commented out the controlled stop competency actions and added TODOs - so this test fails. Makes sense to fix the test when changing the actions to Manoeuvre actions.